packages/common/src/resourceHelpers.ts

```diff
@@ -155,6 +155,7 @@ export function copyBlobToStorageItem(
   storageRequestOptions: auth.IUserRequestOptions
 ): Promise<string[]> {
   return new Promise<string[]>((resolve, reject) => {
+    console.warn("========== TODO ==========");
```

Comment:
Any need for me to mention when these TODOs come through a PR?

Reply:
I don't think so; the goal would simply be to get to 0 "pending specs"/"skipped tests"